src/main/java/practice/kadai2209_7th/AirportController.java

```diff
@@ -18,115 +19,161 @@
 @RestController
 @RequestMapping("/airports")
 public class AirportController {
+
+    List<String> airportCodeList = new ArrayList<>();
+
     Map<String, List<String>> yourAirportMap = new HashMap<>();
 
-    private boolean hasAirportList = false;
+    private final String notFoundMessage = "not found";
+
     @Autowired
     private PortNumber portNum;
 
     @Autowired
     private AirportService service;
 
-    @GetMapping("/airports")
-    public String getAirportList() {
+    AirportAllData airportAllData = new AirportAllData();
 
-        if (!hasAirportList) {
 
-            yourAirportMap = service.getAllDataMap();
+    @ResponseStatus(code = HttpStatus.OK)
+    @GetMapping("/airports")
+    public Map<String, Object> getAirportList() {
```

Comment:
Objectは何でもありのクラスなので濫用すると静的型付け言語であるJavaのメリットが損なわれます。
なるだけ使わないようにしましょう。
```suggestion
    public Map<String, String> getAirportList() {
```

Reply:
ご指摘の個所を見直しました。
1bd32d5d43cd48993c7baf058def8db349f79ae6
![image](https://user-images.githubusercontent.com/113277395/199868421-a9f778c1-1f88-4865-9ea1-829c31e28a2c.png)